app/models/miq_report_result.rb

```diff
@@ -341,4 +330,15 @@ def self.delete_by_userid(userids)
   def self.auto_generated
     where.not(:report_source => "Generated by user")
   end
+
+  private
+
+  def in_user_timezone(&block)
```

Comment:
This seems like a lot of work just to get a timezone to be formatted. Is there a reason we can't just pass in the appropriate time zone to `format_timezone` instead of doing the old switcheroo here?